semversioner/core.py

```diff
@@ -101,6 +101,17 @@ def release(self) -> Release:
 
         return Release(version=next_version_number, changes=changes)
 
+    def next_version(self) -> str:
+        changes: List[Changeset] = self.fs.list_changesets()
+        current_version_number = self.get_last_version()
+        next_version_number = self.get_next_version(changes, current_version_number)
+
+        if next_version_number is None:
+            click.secho("Error: No changes to release. Skipping release process.", fg='red')
+            sys.exit(-1)
+
+        click.echo("Releasing version: %s -> %s" % (current_version_number, next_version_number))
```

Comment:
I think we shouldn't echo any message similar to `Releasing version: %s -> %s` as users won't be able to parse it easily. The main idea of the `next-version` command is to return only the version itself, rather than the whole text. We already have the `semversioner status` command today that does similar thing.

Reply:
This method doesn't seem to return anything.

Reply:
Agreed. Returning _just_ the next version number is preferable.